iob_soc.py

```diff
@@ -128,6 +132,8 @@ def _create_submodules_list(cls, extra_submodules=[]):
                 (iob_tasks, {"purpose": "simulation"}),
                 # Software modules
                 printf,
+                (iob_cache, {"purpose": "simulation"}),
```

Comment:
for some reason the setup only worked if iob_cache was puted down on the submodule_list